torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10881,7 +10881,7 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            gradcheck_nondet_tol=GRADCHECK_NONDET_TOL,
            sample_inputs_func=sample_inputs_adaptive_avg_pool2d),
     OpInfo('nn.functional.adaptive_avg_pool3d',
-           dtypes=floating_types_and(torch.half),
+           dtypes=floating_types_and(torch.half, torch.bfloat16),
```

Comment:
What about the actual tests that `adaptive_avg_pool3d` is computing the correct thing, however?

Because this OpInfo doesn't define a NumPy reference, it doesn't check that the forward is correct.

Reply:
(One option would be to review the current tests AND add a NumPy reference)